### PR TITLE
Update Feature Changelist v2

### PR DESCRIPTION
**Pull request by inc1t3Ful**
++ added recommendations for upgrades
++ added instructions on updating to v2
++ added enable_snat to respective resources
++ added instructions for deprecation of the transit and spoke_vpc

### CODE REVIEW

File: website/docs/guides/feature-changelist-v2.html.md [hunk 1]
```diff
@@ -27,17 +31,20 @@ Otherwise, this list does not really apply. Please view the below list for detai
 |(changed) | --   | policy: allow_deny | **Yes**; renamed to ```action``` |
 |(changed) | --   | policy: log_enable | **Yes**; renamed to ```log_enabled``` |
 |(changed) | fqdn       | fqdn_status       | **Yes**; renamed to ```fqdn_enabled``` |
-|(changed) | gateway    | vpc_size          | **Yes**; change any gateway's ```vpc_size``` to ```gw_size``` |
-|(changed) | gateway    | vpc_net           | **Yes**; change any gateway's ```vpc_net``` to ```subnet``` |
+|(changed) | gateway    | vpc_size          | **Yes**; change any gateways' ```vpc_size``` to ```gw_size``` |
+|(changed) | gateway    | vpc_net           | **Yes**; change any gateways' ```vpc_net``` to ```subnet``` |
+|(changed) | gateway    | enable_nat        | **Yes**; change any gateways' ```enable_nat``` to ```enable_snat``` |
 |(changed) | spoke_vpc  | vpc_size          | **Yes**; change any spoke gateway's ```vpc_size``` to ```gw_size``` |
+|(new)     | spoke_gateway    | enable_snat        | **Yes**; change any **spoke_vpc** resources' ```enable_nat``` to ```enable_snat``` <br><br> Please see **Resource Renaming** table for full instructions |
```
Comment (arvindsree) [inline website/docs/guides/feature-changelist-v2.html.md]: spoke_gateway is a new resource right? they dont need to change

Comment (arvindsree) [reply website/docs/guides/feature-changelist-v2.html.md]: also spoke_vpc and transit_vpc dont have changes right

File: website/docs/guides/feature-changelist-v2.html.md [hunk 2]
```diff
@@ -46,17 +53,17 @@ Otherwise, this list does not really apply. Please view the below list for detai
...
 |(changed) | --         | enable_ldap       | **Yes**; see above for details |
 |(changed) | --         | single_az_ha      | **Yes**; see above for details |
 |(changed) | --         | allocate_new_eip  | **Yes**; see above for details |
 |(changed) | site2cloud | ha_enabled        | **Yes**; see above for details |
-|(changed) | spoke_vpc / spoke_gateway | enable_nat | **Yes**; see above for details |
-|(changed) | --         | single_az_ha      | **Yes**; see above for details |
-|(changed) | transit_vpc / transit_gateway | enable_nat | **Yes**; see above for details |
-|(changed) | --         | connected_transit | **Yes**; see above for details |
-|(changed) | tunnel     | enable_ha         | **Yes**; see above for details |
+|(changed) | spoke_vpc / spoke_gateway | enable_nat | **Yes**; Accepted values are changed to **true** or **false**. Please note this attribute was also renamed. See **Attribute Renaming** table above <br><br> Please see **Resource Renaming** table for full instructions |
```
Comment (arvindsree) [inline website/docs/guides/feature-changelist-v2.html.md]: this isnt changed right?

File: website/docs/guides/feature-changelist-v2.html.md (hunk 1 above)
Comment (inc1t3Ful) [reply website/docs/guides/feature-changelist-v2.html.md]: Yes it is a new resource, it doesnt need a change, but that's why its listed as (new). 

While spoke_vpc and transit_vpc dont have changes per se, its deprecated and the necessary change is that you need to update it to spoke_gateway or transit_gateway respectively and also heed the actual attribute changes on those new resources, and then import. Those steps were explicitly written in the Resource Renaming table.

Is there a better way to document that?

File: website/docs/guides/feature-changelist-v2.html.md (hunk 2 above)
Comment (inc1t3Ful) [reply website/docs/guides/feature-changelist-v2.html.md]: No changes technically. See above comment

**Pull request merged**